precompiles/staking/tx.go

```diff
@@ -43,6 +45,52 @@ const (
 	CancelUnbondingDelegationAuthz = stakingtypes.AuthorizationType_AUTHORIZATION_TYPE_CANCEL_UNBONDING_DELEGATION
 )
 
+// CreateValidator performs create validator.
+func (p Precompile) CreateValidator(
+	ctx sdk.Context,
+	origin common.Address,
+	contract *vm.Contract,
+	stateDB vm.StateDB,
+	method *abi.Method,
+	args []interface{},
+) ([]byte, error) {
+	msg, delegatorHexAddr, err := NewMsgCreateValidator(args, p.stakingKeeper.BondDenom(ctx))
+	if err != nil {
+		return nil, err
+	}
+
+	p.Logger(ctx).Debug(
+		"tx called",
+		"method", method.Name,
+		"args", fmt.Sprintf(
+			"{ min_self_delegation: %s, delegator_address: %s, validator_address: %s, pubkey: %s, value: %s }",
+			msg.MinSelfDelegation.String(),
+			delegatorHexAddr,
+			msg.ValidatorAddress,
+			msg.Pubkey.String(),
+			msg.Value.Amount.String(),
+		),
```

Comment:
please split the args into separate key values

Reply:
done